go/vt/vtctl/vtctl.go

```diff
@@ -489,7 +489,7 @@ var commands = []commandGroup{
 		"Workflow", []command{
 			{"Workflow", commandWorkflow,
```

Comment:
I think it would be a nice touch to be able to tag workflows directly as they're created with `Reshard/MoveTables` as well

Reply:
I wanted to keep the code footprint small to start with : otherwise we need to modify MoveTables/Reshard/Materialize for both v1 and v2 versions. In any case we would need the Workflow command for those who want to tag or retag it later. Is it usable for you with the current implementation?

Reply:
Yeah this is usable now; okay to come back to this later!